pkg/commands/clusterstore/add.go

```diff
@@ -62,21 +76,16 @@ kp clusterstore add my-store -b ../path/to/my-local-buildpackage.cnb`,
 
 	cmd.Flags().StringArrayVarP(&buildpackages, "buildpackage", "b", []string{}, "location of the buildpackage")
 	commands.SetDryRunOutputFlags(cmd)
-	commands.SetTLSFlags(cmd, &factory.TLSConfig)
+	commands.SetTLSFlags(cmd, &tlsCfg)
 	return cmd
 }
 
 func update(store *v1alpha1.ClusterStore, buildpackages []string, factory *clusterstore.Factory, ch *commands.CommandHelper, cs k8s.ClientSet) error {
-	repo, err := k8s.DefaultConfigHelper(cs).GetCanonicalRepository()
-	if err != nil {
-		return err
-	}
-
-	if err = ch.PrintStatus("Adding Buildpackages..."); err != nil {
+	if err := ch.PrintStatus("Adding To ClusterStore..."); err != nil {
```

Comment:
Trivial.. I feel `Adding Buildpackages to ClusterStore...` would be more accurate than `Adding Buildpackages...` or `Adding To ClusterStore...`
whatever you chose to go with, can we avoid the `To` and opt for `to`? no reason to capitalize it :shrug:

Reply:
I felt like `Adding Buildpackages` was misleading in the case of a dry run

> can we avoid the To and opt for to? no reason to capitalize it

makes sense for consistency!

Reply:
This is how it would have looked like for a `--dry-run`. The `(dry-run)` is added to whatever message you give it.
 
```
$ kp csr add default -b gcr.io/paketo-buildpacks/bundler@sha256:af01c3b117e10073228f047f111650b9e777b4cf01bf5474ae3b44f0ba6ac2e1 -b gcr.io/paketo-buildpacks/debug@sha256:802167dab78dd6e999606499faed63766c70cc71669253c6871ae5eda6bae880 --dry-run
**Adding Buildpackages... (dry run)**
	Uploading 'gcr.io/cf-build-service-dev-219913/ssuresh/not-install/paketo-buildpacks_bundler@sha256:af01c3b117e10073228f047f111650b9e777b4cf01bf5474ae3b44f0ba6ac2e1'
	Buildpackage already exists in the store
	Uploading 'gcr.io/cf-build-service-dev-219913/ssuresh/not-install/paketo-buildpacks_debug@sha256:802167dab78dd6e999606499faed63766c70cc71669253c6871ae5eda6bae880'
	Buildpackage already exists in the store
ClusterStore "default" updated (no change)
```

Reply:
I think logging `Uploading 'gcr.io/cf-build-service-dev-219913/ssuresh/not-install/paketo-buildpacks_bundler@sha256:af01c3b117e10073228f047f111650b9e777b4cf01bf5474ae3b44f0ba6ac2e1'` is misleading even if `dry-run` is called out.

One thing that may be missing is we don't mention which buildpackage already exists in the store in the case of a dry-run

Reply:
> One thing that may be missing is we don't mention which buildpackage already exists in the store in the case of a dry-run

it actually does. the output above is for a `--dry-run`. updated the command to be more readable

Reply:
this is how it look now vs [before](https://github.com/vmware-tanzu/kpack-cli/pull/105#discussion_r509692847) 
```
$  kp csr add default \
 -b gcr.io/paketo-buildpacks/bundler@sha256:af01c3b117e10073228f047f111650b9e777b4cf01bf5474ae3b44f0ba6ac2e1 \
 -b gcr.io/paketo-buildpacks/debug@sha256:802167dab78dd6e999606499faed63766c70cc71669253c6871ae5eda6bae880 \
 --dry-run
Adding to ClusterStore... (dry run)
	Buildpackage already exists in the store
	Buildpackage already exists in the store
ClusterStore "default" updated (dry run)
```

Reply:
I think the solution to this logging problem is to log like this

```
Adding to ClusterStore... (dry run)
	Buildpackage gcr.io/paketo-buildpacks/bundler@sha256:af01c3b117e10073228f047f111650b9e777b4cf01bf5474ae3b44f0ba6ac2e1 already exists in the store
	Buildpackage gcr.io/paketo-buildpacks/debug@sha256:802167dab78dd6e999606499faed63766c70cc71669253c6871ae5eda6bae880 already exists in the store
```